client/src/components/PreviousTripsCard/PreviousTripsCard.css

```diff
@@ -0,0 +1,99 @@
+.PreviousTripsCard__container {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  font-family: var(--default-font);
+}
+.PreviousTripCard__container h1 {
+  color: var(--white);
+  width: 50%;
+  padding-left: 5%;
+}
+.PreviousTripCard__trips {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 90%;
+  height: 100%;
+  margin-left: 3%;
+}
+
+.PreviousTripCard__arrow {
+  position: absolute;
+  right: -1%;
+  top: 53%;
+  background-color: var(--yellow-dark);
+  width: 35px;
+  height: 50px;
+  clip-path: polygon(0 0, 80% 50%, 0 100%, 0 80%, 45% 50%, 0 20%);
+  -webkit-box-shadow: 6px 7px 5px -2px rgba(0, 0, 0, 0.64);
+  -moz-box-shadow: 6px 7px 5px -2px rgba(0, 0, 0, 0.64);
+  box-shadow: 6px 7px 5px -2px rgba(0, 0, 0, 0.64);
+  transition: all 0.1s ease-in-out;
+  cursor: pointer;
+  z-index: 2;
+}
+.PreviousTripCard__arrow:hover {
+  transform: scale(1.1);
+  background-color: var(--yellow);
+}
+.PreviousTripCard__trips a {
+  color: var(--white);
+}
+
+.PreviousTripCard__no-trips {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 80%;
+  height: 30%;
+  text-align: center;
+  margin-left: 8%;
+}
+
+.PreviousTripCard__no-trips h2 {
+  margin: 3% 20% 0px 24%;
+
+  font-size: 22px;
+  color: var(--white);
+  display: inline-block;
+}
+.PreviousTripCard__no-trips a {
+  font-size: 18px;
+  top: 150px;
+  margin: 0px 27% 0px 32%;
+
+  color: var(--white);
+  display: inline-block;
+  transition: all 0.1s ease-in-out;
+}
+.PreviousTripCard__no-trips a:hover {
+  transform: scale(1.1);
+  color: var(--yellow);
+}
+
+@media only screen and (max-width: 576px) {
```

Comment:
You did a really nice job starting on responsive here!! For the teeny weeny and the small screen, some of the content is overlapping. Continue working on this to make it look less crowded.

<img width="353" alt="Screen Shot 2019-12-12 at 2 23 46 PM" src="https://user-images.githubusercontent.com/38398204/70742332-40e71600-1ceb-11ea-8a86-796d144013aa.png">